extensions/multimonitor/src/getCommandsModule.ts

```diff
@@ -0,0 +1,31 @@
+import { ServicesManager, CommandsManager, ExtensionManager } from '@ohif/core';
+
+import applyMultiMonitor from './MultiMonitor';
```

Comment:
I'm not a fan of `applyMultiMonitor` import from a folder called `MultiMonitor` which only has a `index.tsx`